src/widgetastic_patternfly/__init__.py

```diff
@@ -709,7 +710,8 @@ def other_div_displayed():
                 self.logger.debug('finishing the menu selection by clicking on %s', level)
                 # No safety check because previous command did it
                 self.browser.click(link, ignore_ajax=True, check_safe=False)
-                self.browser.handle_alert(wait=2.0, squash=True)
+                if handle_alert:
```

Comment:
If `handle_alert` is a `kwarg`

```suggestion
                if kwargs.get('handle_alert', True):
```